.golangci.yml

```diff
@@ -1,10 +1,30 @@
 run:
   timeout: 10m
   skip-dirs:
-    - swagger-gen
+    - cmd/collector/app/sanitizer/cache/mocks
```

Comment:
No support for wildcards, like **/mocks?

Reply:
There is https://golangci-lint.run/usage/configuration/#run-configuration but `**/mocks` is failing

Reply:
It seems like mocks alone is sufficient